.github/workflows/elixir.yml

```
@@ -18,83 +18,60 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    # - name: Set up Elixir v1.13
-    #   uses: erlef/setup-beam@v1
-    #   with:
-    #     elixir-version: '1.13.4' # Define the elixir version [required]
-    #     otp-version: '24.X' # Define the OTP version [required]
-    # - name: Restore dependencies cache
-    #   uses: actions/cache@v3
-    #   with:
-    #     path: deps
-    #     key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
-    #     restore-keys: ${{ runner.os }}-mix-
-    # - name: Install dependencies
-    #   run: mix deps.get
-    # - name: Run tests
-    #   run: mix test
+      - uses: actions/checkout@v3
```

To silence NodeJS warnings 
```suggestion
      - uses: actions/checkout@v4
```